docs/source/specification.md

```diff
@@ -72,3 +78,35 @@ A real project folder may look like:
             └── histology/
                 └── cell_counts.csv
 ```
+
+## File Naming Conventions
+
+File names are *required* to be formatted as set of key-value pairs ending with an *optional* suffix and *required* file extension (e.g. `ses-001_date-20220516_retinotopy.ap.bin`). 
+
+Key-value pairs are separated by underscores while the key and values 
+separated by hypens (e.g. `sub-001_key1-value1_key2-value2.csv`). Key-value pairs other than the *required* `sub` or `ses` are *optional* and can contain 
+additional information. These might include the task name, or run number e.g. `sub-001_task-escape_run-001.csv`
+
+The filename may also end in an *optional* suffix, indicating a logical grouping to which it belongs (e.g. `sub-001_run-001_lightsoff.csv`). 
```

Comment:
I think a better example than "lightsoff" would be "retinotopy" (as in the example project) or perhaps "video", "eyetracking". I understand the the suffix as indicating a subtype of the datatype.
If I had "lightsoff" in my data, I would be more inclined to mark it as "condition-lightsoff" or sth like that